packages/client/components/AddTeamMemberModal.tsx

```diff
@@ -175,6 +176,10 @@ const AddTeamMemberModal = (props: Props) => {
 
   const sendInvitations = () => {
     if (invitees.length === 0) return
+    SendClientSegmentEventMutation(atmosphere, 'Clicked Send Invitation Button', {
```

Comment:
We should follow [Segment naming convention](https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/) (Objection-action framework), so let's rename the event name to "Send Invitation Button Clicked"